packages/server/assets/comments/typedefs/comments.gql

```diff
@@ -1,5 +1,6 @@
 extend type Query {
```

Comment:
Note: park for now; needs better specc'ing and clarifications coming from the UX side